apps/test/unit/templates/teacherDashboard/TeacherDashboardHeaderTest.js

```diff
@@ -94,8 +94,10 @@ describe('TeacherDashboardHeader', () => {
     expect(dropdownLinks.at(1).contains(checkmarkIcon)).to.equal(false);
   });
 
-  it('renders button to edit section details', () => {
-    const wrapper = shallow(<TeacherDashboardHeader {...DEFAULT_PROPS} />);
+  it('renders button to old edit section details', () => {
+    const wrapper = shallow(
+      <TeacherDashboardHeader {...DEFAULT_PROPS} userId={99} />
```

Comment:
magic number alert - this one definitely needs the comment for the opposite "userId % 10 != 0 is in the old edit section details"